workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -522,6 +522,8 @@ def build_mount_command(self, params):
                 params['path'] = '//' + params['path']
         elif self.share_mount.mount_type == "LUSTRE":
             command = command.format(protocol="lustre")
+            if params['path'].count('/') > 1 and params['path'].endswith('/'):
+                params['path'] = params['path'][:-1]
```

Comment:
I'm just wondering if it's ok that we have trailing slashes in path for only lustre mounts. If there are some formal reasons for this then ok. Otherwise it would be great to unify the behavior between different storage types.

Please share what you think before implementing it.

Reply:
Sorry, it was a part of a debugging routine, following commits brings more thorough Lustre support.